### ISSUE TITLE
NTR: Non-glucose reducing substance in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8793]
New term request Non-glucose reducing substance in urine (f346): Galactose, lactose, maltose, or fructose  UMLS: 0428588

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000389
- label: Non-glucose reducing substance in urine
- parents: [Abnormal urine carbohydrate level (HP:0031979)](https://hpo.jax.org/app/browse/term/HP:0031979)
- pmid: PMID:17127733
- def: Abnormal result for this screening test for urinary reducing substances including glucose, fructose, galactose, lactose, pentose and maltose, as well as amino acids, cysteine, tyrosine, homogentisic acid, ketone bodies, ascorbic acid and many drugs. This term is applied if the urine stick test for glucose was negative (normal).
- comment: Detection of reducing substances in urine has been a standard laboratory procedure for about 50 yr. It is used as a screening test for inborn errors of carbohydrate metabolism. Although the test has poor specificity and most states perform mandatory newborn screening for the common genetic defects, most clinical laboratories still perform this as a reflex test on all pediatric urine samples.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8793
- synonyms:
